src/main/java/io/cdap/plugin/MongoDBConfig.java

```diff
@@ -76,21 +76,28 @@ public class MongoDBConfig extends PluginConfig {
   @Nullable
   private String password;
 
+  @Name(MongoDBConstants.CONNECT_USING_SRV_STRING)
+  @Description("Toggle to determine whether to use an SRV connection string for MongoDB. It can be " +
+    "enabled if the MongoDB deployment supports SRV DNS records for connection resolution.")
+  @Nullable
+  private boolean connectUsingSRVString;
```

Comment:
I see this condition below to check this plugin property whether it contains macro or not, but the plugin property is not annotated with @Macro annotation.